javapoet/src/main/java/com/palantir/javapoet/TypeSpec.java

```diff
@@ -459,6 +475,28 @@ void emit(CodeWriter codeWriter, String enumName, Set<Modifier> implicitModifier
         }
     }
 
+    /**
+     * Emits the constructor parameters for a record type. i.e. {@code (String name, int count)}.
+     */
```

Comment:
This comment is unnecessary - it doesn't provide any additional context that is not already obvious from the method signature.

```suggestion
```